examples/retinanet/main.py

```diff
@@ -0,0 +1,71 @@
+"""Main file for running the example.
+
+This file is intentionally kept short. The majority for logic is in libraries
+than can be easily tested and imported in Colab.
```

Comment:
Adding a Colab loading example code and visualizing  preprocessing/results/evaluation would be awesome !

Reply:
Luckily I have something like this. I'll make sure to add it in the documentation such that people can see it.